Testing/functional/tests/cypress/integration/ui/covid19/authenticatedPcrTest.js

```diff
@@ -22,6 +29,14 @@ const processedBanner = "[data-testid=alreadyProcessedBanner]";
 
 describe("Authenticated Pcr Test Registration", () => {
     beforeEach(() => {
+        setupPatientIntercept();
+        setupUserProfileIntercept();
+        setupCommunicationIntercept();
+        setupCommunicationIntercept({
+            communicationType: CommunicationType.InApp,
+            communicationFixture: CommunicationFixture.InApp,
+        });
```

Comment:
Since this is being repeated in many places, I suggest adding a way to set up the standard behaviour in a single call via something like `setupStandardIntercepts()`.  It could either be parameterless and simply used in place of performing these 4 specific calls, or it could have an optional parameter object that would allow you to choose which intercepts to apply and which to override with different fixtures when the defaults don't suffice.

Reply:
Changed.